apps/api/src/app/events/usecases/add-job/should-add-digest-job.usecase.ts

```diff
@@ -1,28 +0,0 @@
-import { Injectable } from '@nestjs/common';
-import { JobRepository, JobStatusEnum } from '@novu/dal';
-import { StepTypeEnum } from '@novu/shared';
-import { AddJobCommand } from './add-job.command';
-
-@Injectable()
-export class ShouldAddDigestJob {
-  constructor(private jobRepository: JobRepository) {}
-
-  public async execute(command: AddJobCommand): Promise<boolean> {
-    const data = await this.jobRepository.findById(command.jobId);
-
-    const isValidDigestStep = data.type === StepTypeEnum.DIGEST && data.digest.amount && data.digest.unit;
-    if (!isValidDigestStep || !data) {
-      return true;
```

Comment:
Arbitrarily I decided to remove this use case and integrate the logic in the `AddDigestJob` use case to be able to simplify the code. We were hitting the database extra times without any need (in my opinion for this flow) and therefore having to do many validations that were already checked, just because in the case this use case was used standalone.